.eslintrc.json

```diff
@@ -32,7 +32,7 @@
         "@typescript-eslint/no-var-requires": 0,
         "@typescript-eslint/no-explicit-any": 0,
         "@typescript-eslint/no-unused-vars": [
-            "error",
+            "warn",
```

Comment:
Did you intend to check-in this change?

Reply:
ype. Local we don't need to fail when you have an extra import